neural_modelling/src/neuron/input_types/input_type.h

```diff
@@ -10,17 +10,18 @@ typedef struct input_type_t* input_type_pointer_t;
 //! \param[in] value The value of the input before scaling
```

Comment:
You might want to change this to \param[in/out] and update the comment

Reply:
Done